src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/HelperSignature.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Internal.Text;
+
+namespace ILCompiler.DependencyAnalysis.ReadyToRun
+{
+    public class ReadyToRunHelperSignature : Signature
+    {
+        private ReadyToRunHelper _helper;
+
+        public ReadyToRunHelperSignature(ReadyToRunHelper helper)
+        {
+            _helper = helper;
+        }
+
+        protected override int ClassCode => 208107954;
+
+        public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
+        {
+            ObjectDataBuilder dataBuilder = new ObjectDataBuilder();
```

Comment:
Nit: depending on how much you care about perf, in cases like this it might be better to just do:

```csharp
return new ObjectData(
    new[] { (byte)ReadyToRunFixupKind.READYTORUN_FIXUP_Helper, (byte)_helper },
    Array.Empty<Relocation>(),
    1,
    new ISymbolDefinitionNode[] { this });
```

`ObjectDataBuilder` is nice if you need to build more complicated data structures, but if all you need is two bytes and a symbol, this saves you two array re-allocations.

Reply:
Fixed - thanks Michal!